runtime/codert_vm/xnathelp.m4

```diff
@@ -988,6 +990,7 @@ ifdef({OMR_GC_CONCURRENT_SCAVENGER},{
 END_PROC(jitReadBarrier)
 
 START_PROC(jitReferenceArrayCopy)
+	pop uword ptr J9TR_VMThread_jitReturnAddress[_rbp]
```

Comment:
This should probably go after the `FASTCALL_EXTERN`.  It makes more sense (at least to me) to group the executable code together.

Reply:
Good suggestion, it has been updated.

Reply:
Good point, I've moved it outside. Though I don't think it would cause any failures on other platforms as this M4 file is only consumed by X86.

Reply:
Yep - vacation brain :)